### PR TITLE
Add AlmaLinux 9 support

### PR DESCRIPTION
**Pull request by pasimko**

### CODE REVIEW

File: kitchen.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 driver:
   name: vagrant
+  flavor_ref: "m1.medium"
```
Comment (ramereth) [inline kitchen.yml]: This doesn't do anything for vagrant. Please remove.

**Review by ramereth — CHANGES_REQUESTED** [review]
Please also update the openstack and dokken kitchen files.

File: kitchen.yml (hunk 1 above)
Comment (pasimko) [reply kitchen.yml]: Hmm. I get OOM without that line. ```[  459.325324] Out of memory: Killed process 2138 (platform-python) total-vm:571200kB, anon-rss:520324kB, file-rss:128kB, shmem-rss:0kB, UID:0 pgtables:1160kB oom_score_adj:0```

Comment (ramereth) [reply kitchen.yml]: This is likely due to the fact we're in this transition between how we're using the kitchen CLI. For now, please go ahead and remove it and then update the other two kitchen file. Makes sure you add this in the openstack file.

**Review by ramereth — APPROVED** [review]
Patch bump

**Comment by pasimko** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.4.1.  Have a nice day!

**Pull request merged**